variable/technology/technologies.yaml

```
@@ -28,7 +76,21 @@ Capacity|Electricity|Hydro:
    description: Total installed electricity generation capacity of hydro power
                 plants
    unit: GW
+   
+Capacity|Electricity|Hydro|Reservoir:
+   description: Total installed electricity generation capacity of hydro power plants with 
+                reservoirs with seasonal storage potential
+   unit: GW
 
+Capacity|Electricity|Hydro|Pumped Storage:
+   description: Total installed electricity generation capacity of hydro power plants with 
+                pumped storage
+   unit: GW
+   
+Capacity|Electricity|Hydro|Run of River:
```

I can't say why, but this sounds more "correct" to me...

```suggestion
Capacity|Electricity|Hydro|Run-of-River:
```